lua/ui/uiutil.lua

```diff
@@ -1492,3 +1492,68 @@ function CreateHorzFillGroup(parent, filename)
     group._right = right
     return group
 end
+
+---@class SCXMenuPanelBorderTable
+---@field tl Bitmap
+---@field tm Bitmap
+---@field tr Bitmap
+---@field l Bitmap
+---@field r Bitmap
+---@field bl Bitmap
+---@field bm Bitmap
+---@field br Bitmap
+
+---@param parent Control
+---@return SCXMenuPanelBorderTable
+function CreateSCXMenuPanelBorder(parent)
+    ---@type SCXMenuPanelBorderTable
```

Comment:
Assigning the class alongside the actual table would make the annotations more concise.
```suggestion
    ---@class SCXMenuPanelBorderTable
```

Reply:
You also have to add 
```
---@overload fun(parent: Control, filename: Lazy<FileName>, debugname: string?): Bitmap
````
to the Bitmap class for intellisense to get the correct type of the table values.